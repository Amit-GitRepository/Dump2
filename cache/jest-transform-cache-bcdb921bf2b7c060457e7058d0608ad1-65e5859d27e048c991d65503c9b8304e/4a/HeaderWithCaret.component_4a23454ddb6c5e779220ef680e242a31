Object.defineProperty(exports,"__esModule",{value:true});var cov_fuae9r4ue=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/HeaderWithCaret.component.js',hash='216a141c5e8caae787351c693b79ffaacf816c97',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/HeaderWithCaret.component.js',statementMap:{'0':{start:{line:13,column:83},end:{line:13,column:93}},'1':{start:{line:14,column:4},end:{line:24,column:15}},'2':{start:{line:27,column:0},end:{line:33,column:2}},'3':{start:{line:34,column:0},end:{line:41,column:2}}},fnMap:{'0':{name:'(anonymous_0)',decl:{start:{line:12,column:2},end:{line:12,column:3}},loc:{start:{line:12,column:12},end:{line:25,column:3}},line:12}},branchMap:{'0':{loc:{start:{line:17,column:11},end:{line:21,column:17}},type:'cond-expr',locations:[{start:{line:17,column:25},end:{line:19,column:17}},{start:{line:19,column:20},end:{line:21,column:17}}],line:17}},s:{'0':0,'1':0,'2':0,'3':0},f:{'0':0},b:{'0':[0,0]},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _createClass=function(){function defineProperties(target,props){for(var i=0;i<props.length;i++){var descriptor=props[i];descriptor.enumerable=descriptor.enumerable||false;descriptor.configurable=true;if("value"in descriptor)descriptor.writable=true;Object.defineProperty(target,descriptor.key,descriptor);}}return function(Constructor,protoProps,staticProps){if(protoProps)defineProperties(Constructor.prototype,protoProps);if(staticProps)defineProperties(Constructor,staticProps);return Constructor;};}();

var _propTypes=require('prop-types');var _propTypes2=_interopRequireDefault(_propTypes);
var _react=require('react');var _react2=_interopRequireDefault(_react);
var _HeaderWithCaret=require('./HeaderWithCaret.styles');var _HeaderWithCaret2=_interopRequireDefault(_HeaderWithCaret);
var _Shared=require('../../../Shared');
var _constants=require('../../../../themes/constants.styles');
var _application=require('../../../../themes/application.styles');
var _reactNative=require('react-native');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}function _classCallCheck(instance,Constructor){if(!(instance instanceof Constructor)){throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self,call){if(!self){throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}return call&&(typeof call==="object"||typeof call==="function")?call:self;}function _inherits(subClass,superClass){if(typeof superClass!=="function"&&superClass!==null){throw new TypeError("Super expression must either be null or a function, not "+typeof superClass);}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass;}var

HeaderWithCaret=function(_Component){_inherits(HeaderWithCaret,_Component);function HeaderWithCaret(){_classCallCheck(this,HeaderWithCaret);return _possibleConstructorReturn(this,(HeaderWithCaret.__proto__||Object.getPrototypeOf(HeaderWithCaret)).apply(this,arguments));}_createClass(HeaderWithCaret,[{key:'render',value:function render()
{cov_fuae9r4ue.f[0]++;var _ref=(cov_fuae9r4ue.s[0]++,
this.props),headerColor=_ref.headerColor,iconSize=_ref.iconSize,children=_ref.children,style=_ref.style,customIconStyle=_ref.customIconStyle,caretShadow=_ref.caretShadow;cov_fuae9r4ue.s[1]++;
return(
_react2.default.createElement(_reactNative.View,{style:_HeaderWithCaret2.default.container},
_react2.default.createElement(_Shared.BottomCaretView,{caretColor:headerColor,iconSize:iconSize,customIconStyle:customIconStyle},
caretShadow?(cov_fuae9r4ue.b[0][0]++,_react2.default.createElement(_reactNative.View,{elevation:2,style:[_HeaderWithCaret2.default.wrapper,style,{backgroundColor:headerColor},_application.shadowActiveStyle]},
children)):(cov_fuae9r4ue.b[0][1]++,
_react2.default.createElement(_reactNative.View,{style:[_HeaderWithCaret2.default.wrapper,style,{backgroundColor:headerColor}]},
children)))));




}}]);return HeaderWithCaret;}(_react.Component);cov_fuae9r4ue.s[2]++;

HeaderWithCaret.defaultProps={
headerColor:_constants.colors.SECONDARY_TMOVE,
iconSize:30,
customIconStyle:{},
style:{},
caretShadow:false};cov_fuae9r4ue.s[3]++;

HeaderWithCaret.propTypes={
headerColor:_propTypes2.default.string,
iconSize:_propTypes2.default.number,
customIconStyle:_propTypes2.default.object,
children:_propTypes2.default.node,
style:_propTypes2.default.object,
caretShadow:_propTypes2.default.bool};exports.default=

HeaderWithCaret;