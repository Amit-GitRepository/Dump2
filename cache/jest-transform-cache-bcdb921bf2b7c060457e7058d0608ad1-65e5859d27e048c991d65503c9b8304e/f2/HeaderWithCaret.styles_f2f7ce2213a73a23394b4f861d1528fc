Object.defineProperty(exports,"__esModule",{value:true});var cov_2o1364wi5g=function(){var path="/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/HeaderWithCaret.styles.js",hash="fb7646488b272eb67c9880635df1aad2d6ac84eb",global=new Function('return this')(),gcv="__coverage__",coverageData={path:"/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/HeaderWithCaret.styles.js",statementMap:{},fnMap:{},branchMap:{},s:{},f:{},b:{},_coverageSchema:"332fd63041d2c1bcb487cc26dd0d5f7d97098a6c"},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();exports.default=
{
wrapper:{
padding:10,
borderTopLeftRadius:10,
borderTopRightRadius:10}};