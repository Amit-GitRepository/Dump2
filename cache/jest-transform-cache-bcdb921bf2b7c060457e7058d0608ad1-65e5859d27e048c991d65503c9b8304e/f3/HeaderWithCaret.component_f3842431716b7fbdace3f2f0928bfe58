Object.defineProperty(exports,"__esModule",{value:true});var cov_fuae9r4ue=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/HeaderWithCaret.component.js',hash='51577a4f30d6a54555d6dd5be5883f916a074364',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/HeaderWithCaret.component.js',statementMap:{'0':{start:{line:12,column:70},end:{line:12,column:80}},'1':{start:{line:13,column:4},end:{line:20,column:15}},'2':{start:{line:23,column:0},end:{line:28,column:2}},'3':{start:{line:29,column:0},end:{line:35,column:2}}},fnMap:{'0':{name:'(anonymous_0)',decl:{start:{line:11,column:2},end:{line:11,column:3}},loc:{start:{line:11,column:12},end:{line:21,column:3}},line:11}},branchMap:{},s:{'0':0,'1':0,'2':0,'3':0},f:{'0':0},b:{},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _createClass=function(){function defineProperties(target,props){for(var i=0;i<props.length;i++){var descriptor=props[i];descriptor.enumerable=descriptor.enumerable||false;descriptor.configurable=true;if("value"in descriptor)descriptor.writable=true;Object.defineProperty(target,descriptor.key,descriptor);}}return function(Constructor,protoProps,staticProps){if(protoProps)defineProperties(Constructor.prototype,protoProps);if(staticProps)defineProperties(Constructor,staticProps);return Constructor;};}();

var _propTypes=require('prop-types');var _propTypes2=_interopRequireDefault(_propTypes);
var _react=require('react');var _react2=_interopRequireDefault(_react);
var _HeaderWithCaret=require('./HeaderWithCaret.styles');var _HeaderWithCaret2=_interopRequireDefault(_HeaderWithCaret);
var _Shared=require('../../../Shared');
var _constants=require('../../../../themes/constants.styles');
var _reactNative=require('react-native');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}function _classCallCheck(instance,Constructor){if(!(instance instanceof Constructor)){throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self,call){if(!self){throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}return call&&(typeof call==="object"||typeof call==="function")?call:self;}function _inherits(subClass,superClass){if(typeof superClass!=="function"&&superClass!==null){throw new TypeError("Super expression must either be null or a function, not "+typeof superClass);}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass;}var

HeaderWithCaret=function(_Component){_inherits(HeaderWithCaret,_Component);function HeaderWithCaret(){_classCallCheck(this,HeaderWithCaret);return _possibleConstructorReturn(this,(HeaderWithCaret.__proto__||Object.getPrototypeOf(HeaderWithCaret)).apply(this,arguments));}_createClass(HeaderWithCaret,[{key:'render',value:function render()
{cov_fuae9r4ue.f[0]++;var _ref=(cov_fuae9r4ue.s[0]++,
this.props),headerColor=_ref.headerColor,iconSize=_ref.iconSize,children=_ref.children,style=_ref.style,customIconStyle=_ref.customIconStyle;cov_fuae9r4ue.s[1]++;
return(
_react2.default.createElement(_reactNative.View,{style:_HeaderWithCaret2.default.container},
_react2.default.createElement(_Shared.BottomCaretView,{caretColor:headerColor,iconSize:iconSize,customIconStyle:customIconStyle},
_react2.default.createElement(_reactNative.View,{style:[_HeaderWithCaret2.default.wrapper,style,{backgroundColor:headerColor}]},
children))));



}}]);return HeaderWithCaret;}(_react.Component);cov_fuae9r4ue.s[2]++;

HeaderWithCaret.defaultProps={
headerColor:_constants.colors.SECONDARY_TMOVE,
iconSize:30,
customIconStyle:{},
style:{}};cov_fuae9r4ue.s[3]++;

HeaderWithCaret.propTypes={
headerColor:_propTypes2.default.string,
iconSize:_propTypes2.default.number,
customIconStyle:_propTypes2.default.object,
children:_propTypes2.default.node,
style:_propTypes2.default.object};exports.default=

HeaderWithCaret;