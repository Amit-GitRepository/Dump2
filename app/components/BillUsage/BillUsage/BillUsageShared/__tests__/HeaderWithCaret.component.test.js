import HeaderWithCaret from '../HeaderWithCaret.component';
import React from 'react';
import renderer from 'react-test-renderer';

describe('BillHistoryList component', () => {
  it('renders correctly', () => {
    const component = renderer.create(<HeaderWithCaret />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
