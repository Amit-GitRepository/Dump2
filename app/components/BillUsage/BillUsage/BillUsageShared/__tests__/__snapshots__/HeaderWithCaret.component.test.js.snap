// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BillHistoryList component renders correctly 1`] = `
<View
  style={undefined}
>
  <View>
    <View
      style={Object {}}
    >
      <View
        style={
          Array [
            Object {
              "borderTopLeftRadius": 10,
              "borderTopRightRadius": 10,
              "padding": 10,
            },
            Object {},
            Object {
              "backgroundColor": "#FBBC1E",
            },
          ]
        }
      />
    </View>
    <Text
      accessible={true}
      allowFontScaling={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#FBBC1E",
            "fontSize": 30,
          },
          Array [
            Object {
              "alignSelf": "center",
              "backgroundColor": "transparent",
            },
            Object {
              "bottom": 12,
              "marginBottom": -12,
            },
            Object {},
          ],
          Object {
            "fontFamily": "iservice",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
        ]
      }
    >
      Ó§Å
    </Text>
  </View>
</View>
`;
